src/modules/db_operations/operations.py

```diff
@@ -112,26 +115,43 @@ def drop_and_populate_phenotype_db(app, db, species, reload_files=True):
   logger.info("Loading phenotypes...")
   etl_manager.load_tables(PhenotypeDatabase, species_list=species)
 
+def drop_and_populate_phenotype_metadata(app, db, species, reload_files=True):
```

Comment:
Quick question, is it possible to drop & populate the two phenotype tables separately? If you're using foreign keys in one of them, it might prevent the other table from being cleared correctly.  If you've tested it and it works that's great :)

Reply:
That's a good question. I would weigh on your and @r-vieira decision how we want to handle this. As of now, I think it fails if we won't populate tables in the right order (PhenotypeMetadata, PhenotypeDatabase).

Reply:
Ah, in that case, I'd recommend combining them into a single database operation -- see `drop_and_populate_wormbase_genes` for an example, and the doc comment in `ETLManager.clear_tables` for more info on handling multiple tables with dependencies.  That way, Andersen Lab would just run a single operation that handles the entire rebuild process for them.

Reply:
I'd defer to @r-vieira though, if there's ever a situation where we would want to drop / rebuild them separately?